MantidQt/MantidWidgets/src/DataProcessorUI/GenericDataProcessorPresenter.cpp

```diff
@@ -472,9 +472,8 @@ void GenericDataProcessorPresenter::endReduction() {
 /**
 Handle reduction error
 */
-void GenericDataProcessorPresenter::reductionError(std::exception ex) {
-
-  m_view->giveUserCritical(ex.what(), "Error");
+void GenericDataProcessorPresenter::reductionError(QString ex) {
+  m_view->giveUserCritical(ex.toStdString(), "Error");
```

Comment:
I think we should update the *view* class so that `giveUserCritical` can accept a QString so that we avoid the conversion to `std::string`